mk/localdev.mk

```diff
@@ -83,6 +85,13 @@ restart_django_foreground:
 	$(podman) exec -it osidb-service pkill -f "python3 manage.py runserver"
 	$(podman) exec -it osidb-service python3 manage.py runserver 0.0.0.0:8000
 
+.PHONY : start_gunicorn_fg
+start_gunicorn_fg:
+	@echo ">Note that after CTRL+C, django server will restart in the background, as long as osidb-service is running."
+	@sleep 0.5
+	$(podman) exec -it osidb-service pkill -f "python3 manage.py runserver"
+	$(podman) exec -it osidb-service gunicorn config.wsgi --config gunicorn_config.py
```

Comment:
do we need to be explicit with using config.settings_local here ?

Reply:
nope, that is set at the container level already (notice how we don't do it for `start-local`)